android/quest/src/main/java/org/smartregister/fhircore/quest/ui/login/LoginViewModel.kt

```diff
@@ -213,7 +216,13 @@ constructor(
     } else {
       // Prevent user from logging in with different credentials
       val existingCredentials = secureSharedPreference.retrieveCredentials()
-      if (existingCredentials != null && !username.equals(existingCredentials.username, true)) {
+      if (
+        existingCredentials != null &&
+          !username.equals(
+            existingCredentials.username,
+            true,
+          )
+      ) {
```

Comment:
Have you enabled `ktfmt` for code formatting? Did you run `spotlessApply` to fix issues with code formatting?